training/src/main/java/wolox/training/models/Book.java

```diff
@@ -147,6 +152,8 @@ public Integer getPages() {
     }
 
     public void setPages(Integer pages) {
```

Comment:
To make it more robust, you should also check that the number of pages is greater than 0.